docs/pages/blog/meet-joy.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Meet Joy UI－a new starting point to your design system
+description: Wip description.
+date: 2022-06-01T00:00:00.000Z
+authors: ['danilo-leal', 'siriwatknp']
+tags: ['News']
+card: true
+---
+
+:::warning
+**work in progress**
+:::
+
+If you're a close follower of everything MUI, you might be aware that we've been working on a new design system product. If not, we're writing this piece to give you a first look at Joy－MUI's new starting point for your design system!
+
+## How this is relevant to you?
+
+You might intuitively associate MUI with Material UI, which despite the indeed potentially confusing names, are separate entities. Material UI is MUI's React implementation of Google's Material Design and over the years has established itself as a go-to library for quickly breathing life into products, mostly due to the number of components you get out of the box. However, the components do come with that default 2014 Google look and feel that a good portion of you grew to dislike. And, as we've reconfirmed with our latest developer survey, the design aspect of UI libraries is one of the most important ones whenever evaluating which to go for.
+
+So, if you ever considered going with Material UI because of the thought through developer experience, reliability, and completeness but ended up hesitating because of the design, this is great news for you. Joy is meant to offer you all of that with a simpler and more modern look and feel.
+
+## Why not just build a different theme?
+
+One of the things we're optimizing for with this new product is to break away from Material Design. A different theme would still require you and your team to have at least some sense of how Material Design is structured and how that is expressed into Material UI in order to reach a fresher look. We wanted to provide an alternative to that. With Joy, we want you to be able to start, by default, with a fresher design.
+
+Additionally, we had many ideas for improving the experience of developing design systems and a separate package is an opportunity to prototype and test some of these new features without running the risk of potentially making any mistakes and causing inconveniences to hundreds of teams that use Material UI today.
+
+Material UI and Joy UI will definitely share most of the conventions, features, and overall philosophy. This will not only decrease the learning curve for any of you who are willing to try but also because we think of the two containing our take on the ideal developer experience for developing design systems. So, if at any time, Joy UI has something that Material UI doesn't, the catch-up will at some point happen, and the same goes for the other way around.
```

Comment:
```suggestion
Material UI and Joy UI will still share many conventions and features, and overall philosophy. This will not only decrease the learning curve for adopting Joy, but also because we think of them both representing the ideal developer experience for developing design systems. So if at any time Joy UI has a feature that Material UI doesn't, it will catch-up at some point, and vice versa.
```

> for any of you

There is only one of me. 🙂 

(Write as if addressing one person).